src/com/jambit/STDatabase.java

```diff
@@ -0,0 +1,22 @@
+package com.jambit;
+
+import java.io.IOException;
+import java.sql.SQLException;
+
+public class STDatabase {
+  private static Database database = null;
+
+  private STDatabase() {}
+
+  public static Database getInstance()
+      throws Database.MultipleConnectionException, IOException, SQLException,
+          Database.TimeOutException {
+    if (database == null) {
+      database = new Database();
```

Comment:
Use DatabaseConnection here instead of calling your custom database class. Instead: You call this class from your database class.